src/query.js

```diff
@@ -190,6 +190,46 @@ function _lambdaExpr(var_name, expr) {
   return new Expr({ lambda: wrap(var_name), expr: wrap(expr) });
 }
 
+/**
+ * See the [docs](https://fauna.com/documentation/queries#basic_forms).
+ *
+ * Invokes a given function passing in the provided arguments
+ *
+ * ```
+ * Call(Ref("functions/a_function"), 1, 2)
+ * ```
+ *
+ * @param {module:query~ExprArg} ref
+ * @param {...module:query~ExprArg} args
+ * @return {Expr}
+ * */
+function Call(ref) {
+  arity.min(1, arguments);
+  var args = argsToArray(arguments);
+  args.shift();
+  return new Expr({ call: wrap(ref), arguments: varargs(args) });
+}
+
+/**
+ * See the [docs](https://fauna.com/documentation/queries#basic_forms).
+ *
+ * Constructs a `@query` type using the Lambda() or a function.
+ *
+ * ```
+ * Query(Lambda(['a', 'b'], Add(Var('a'), Var('b'))))
+ * Query(function (a, b) { return Add(a, b) })
+ * ```
+ *
+ * @param {module:query~ExprArg|function} lambda
+ * @return {Expr}
+ * */
+function Query(lambda) {
+  arity.exact(1, arguments);
+  if (lambda instanceof Function)
+    lambda = _lambdaFunc(lambda);
```

Comment:
Why do you need to call `_lambdaFunc` specifically? Seems like `wrap` already does that.